packages/snap-store-mobx/src/Autocomplete/Stores/FacetStore.test.ts

```diff
@@ -26,7 +26,7 @@ describe('Facet store', () => {
 	});
 
 	it('uses the search store FacetStore', () => {
-		const searchData = mockData.autocompleteMeta();
+		const searchData = mockData.searchMeta();
```

Comment:
Since this is in Autocomplete store, we should probably use autocomplete responses.